Alignment/MuonAlignment/plugins/MuonGeometryArrange.cc

```diff
@@ -44,7 +44,10 @@ MuonGeometryArrange::MuonGeometryArrange(const edm::ParameterSet& cfg)
       _levelStrings(cfg.getUntrackedParameter<std::vector<std::string> >("levels")),
       _writeToDB(false),
       _commonMuonLevel(align::invalid),
-      firstEvent_(true) {
+      firstEvent_(true),
+      idealLabel1("MuonGeometryArrangeLabel1"),
+      idealLabel2("MuonGeometryArrangeLabel2"),
+      idealLabel3("MuonGeometryArrangeLabel3") {
```

Comment:
Given the treatments with the three labels are not identical, I suggest using more intention revealing names than `idealLabel`.

Reply:
They are identical ideal geometries. This code needs three ideal geometries, so the labels were ```idealLabel```. 
I tried to use a bit detailed label names but basically the label name comes from ```inputMethod1()```, ```inputMethod2()```, and ```inputMethod3()```.

Reply:
My mistake for not clearly expressing my concern. What I want to emphasis is related to the function `void MuonGeometryArrange::compareGeometries(Alignable* refAli, Alignable* curAli, Alignable* curCopy)`, which clearly shows the meaning of each `Alignable*`s via the argument name. As the three ideal geometry labels has their own role in geometry comparison, could you use names like `refMuonGeometryArrangeLabel` to make the label names less ambiguous?

Reply:
Since all of the input geometries are the same ideal geometry, so I think if I put the label as reference and current, it looks like they are different geometries. So, I would like to keep ```idealLable```.

Reply:
Ok, thank you for the explaination.